src/views/ModelPlan/ReadOnly/GeneralCharacteristics/index.tsx

```diff
@@ -483,15 +483,23 @@ const ReadOnlyGeneralCharacteristics = ({
                       ).split(' - ')[1]
                   )
                   .join(', '),
-                geographiesRegionTypes: geographiesRegionTypes?.map(region => {
-                  return (
-                    <li key={region}>
-                      {generalCharacteristicsT(
-                        `geographiesRegionTypes.options.${region}`
-                      )}
-                    </li>
-                  );
-                }),
+                geographiesRegionTypes:
+                  geographiesRegionTypes?.length !== 0 ? (
+                    geographiesRegionTypes?.map(region => {
+                      return (
+                        <li key={region}>
+                          {generalCharacteristicsT(
+                            `geographiesRegionTypes.options.${region}`
+                          )}
+                        </li>
+                      );
+                    })
+                  ) : (
+                    <em className="text-base">
+                      {generalCharacteristicsMiscT('noAdditionalInformation')}
+                    </em>
```

Comment:
You can just pass null here if there are no values selected and the component will render `No additional information specified` correctly